release_notes/47-release-notes.adoc

```diff
@@ -0,0 +1,348 @@
+
+//ROX-24052
+//tech preview
+[id="generate-sboms-from-the-scanned-container-images_{context}"]
+=== Generate SBOMs from the scanned container images
+
+:FeatureName: Generation of SBOMs from the scanned container images
+include::snippets/technology-preview.adoc[]
+
+With {product-title-short} 4.7, you can now generate Software Bill of Materials (SBOMs) from the scanned container images. These SBOMs give you a detailed overview of all software components, dependencies and libraries within your application.
+{product-title-short} creates SBOMs of the Analyzed type and complies with the Software Package Data Exchange (SPDX) 2.3 specification.
+
+For more information about the SBOMs of the Analyzed type, see
+link:https://www.cisa.gov/sites/default/files/2023-04/sbom-types-document-508c.pdf[Types of SBOM Documents] (America's Cyber Defense Agency documentation).
+
+//ROX-17535
+[id="automatic-certificate-rotation-by-using-the-mtls-architecture_{context}"]
+=== Automatic certificate rotation by using the mTLS architecture
+
+{product-title-short} 4.7 introduces certificate-based architecture (mTLS) for communication between your secured cluster components and Central. 
+This feature enables the automatic rotation of certificates. {product-title-short} components use these certificates on your secured clusters and require no user interaction.
+
+Your secured cluster continues to function even if the initBundle and its bootstrap credentials are revoked, as these certificates are separate from the bootstrap credentials in the initBundle.
+
+//ROX-14616
+[id="protect-and-run-secured-clusters-on-red-openshift-on-ibm-cloud_{context}"]
+=== Protect and run secured clusters on Red{nbsp}OpenShift on IBM Cloud
+
+In {product-title-short} 4.7, you can now protect {product-roibm} and run secured clusters on {product-roibm}.
+
+//ROX-13542
+[id="use-the-short-lived-oidc-credentials-from-microsoft-azure_{context}"]
+=== Use the short-lived OIDC credentials from Microsoft Azure
+
+With {product-title-short} 4.7, you can now use short-lived OpenID Connect (OIDC) credentials. The Microsoft Azure Identity Provider provides these credentials, and you can use these credentials to access the services in Microsoft Azure, such as the Azure Container Registry and Azure Sentinel.
+
+This feature builds on the {product-title-short} 4.4 release, which introduced the ability to use short-lived OIDC credentials from Amazon Web Services (AWS) and Google Cloud Platform (GCP) cloud providers to authenticate to the services hosted on these platforms.
+
+[id="certifications_{context}"]
+== Certifications
+
+{product-title-managed} is certified according to the following global standards for security, compliance, and data protection:
+
+* ISO/IEC 27001:2022
+* ISO/IEC 27017:2015
+* ISO/IEC 27018:2019
+* PCI DSS 4.0
+* SOC 2 Type 2
+* SOC 2 Type 3
+
+[id="notable-technical-changes_{context}"]
+== Notable technical changes
```

Comment:
I think we need another entry in this section about changes to the Collector runtime configuration- 

```
* Collector Runtime Configuration now uses an enum to enable and disable features. For example, `networking.externalIps.enable = true` now becomes `networking.externalIps.enabled = ENABLED`. Valid values are `ENABLED` and `DISABLED`.
```

cc @JoukoVirtanen